pom.xml

```
@@ -11,7 +11,7 @@
 
     <properties>
         <flink.version>1.20.0</flink.version>
```

While we're at it, let's also update the Flink version to 1.20.1
```suggestion
        <flink.version>1.20.1</flink.version>
```